tests/files/test.txt

```diff
@@ -0,0 +1 @@
+test
```

Comment:
Add this new file to use it in the test case is OK? Or should I create a temporary file?